src/models/converters/entries.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const _ = require('lodash')
+
+function convertToInternalEntry (data) {
```

Comment:
I don't see these conversion methods used. I guess they're just included for completeness?